src/Microsoft.Health.Fhir.CosmosDb.Initialization.UnitTests/CosmosDbInitializationTests.cs

```diff
@@ -0,0 +1,69 @@
+// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System;
+using System.Linq;
+using Microsoft.Azure.Cosmos.Scripts;
+using Microsoft.Health.Core.Extensions;
+using Microsoft.Health.Fhir.CosmosDb.Core.Features.Storage.StoredProcedures;
+using Microsoft.Health.Fhir.CosmosDb.Initialization.Features.Storage.StoredProcedures;
+using Xunit;
+
+namespace Microsoft.Health.Fhir.CosmosDb.Initialization.UnitTests
+{
+    public class CosmosDbInitializationTests
```

Comment:
This test class is missing classification.

Please, add the following attributes to the test class:
    [Trait(Traits.OwningTeam, OwningTeam.Fhir)]
    [Trait(Traits.Category, Categories.DataSourceValidation)]

Reply:
done